protocol/indexer/events/update_clob_pair_create.go

```diff
@@ -0,0 +1,24 @@
+package events
+
+import (
+	v1 "github.com/dydxprotocol/v4-chain/protocol/indexer/protocol/v1"
+	"github.com/dydxprotocol/v4-chain/protocol/x/clob/types"
+)
+
+// NewUpdateClobPairEvent creates a UpdateClobPairEventV1
+// representing update of a clob pair.
+func NewUpdateClobPairEvent(
+	clobPairId uint32,
+	status types.ClobPair_Status,
+	quantumConversionExponent int32,
+	subticksPerTick uint32,
+	stepBaseQuantums uint64,
+) *UpdateClobPairEventV1 {
+	return &UpdateClobPairEventV1{
+		ClobPairId:                clobPairId,
+		Status:                    v1.ConvertToClobPairStatus(status),
+		QuantumConversionExponent: quantumConversionExponent,
+		SubticksPerTick:           subticksPerTick,
+		StepBaseQuantums:          stepBaseQuantums,
```

Comment:
Worth adding a comment why you don't include updates to the `metadata` field?

Reply:
This was determined by the indexer `UpdateClobPairEventV1` proto, but I imagine it is because we don't support changing a clob pair's metadata because transitioning from a perpetual to an asset would break things

Reply:
can add a comment though